test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+class UsersControllerTest < ActionDispatch::IntegrationTest
+
```

Comment:
Did not hit.
This empty line is not needed here. Instead, it is needed before the block.